config.json

```diff
@@ -250,6 +250,15 @@
           "numbers"
         ]
       },
+      {
+        "slug": "difference-of-squares",
+        "name": "Difference of Squares",
+        "uuid": "2ffe1999-e0d5-4870-9abf-878046f094b8",
```

Comment:
I didn't check but i assume you generated a unique UUID? (you can't copy the one from JS)

Reply:
I did indeed, via configlet.